lib/cmds/merge_cmds/show.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  AppActionCategoryParams,
+  callAppAction,
+  isResultWithError
+} from '@contentful/app-action-utils'
+import { PlainClientAPI } from 'contentful-management'
+import type { Argv } from 'yargs'
+import {
+  getAppActionId,
+  getAppDefinitionId,
+  Host
+} from '../../utils/app-actions-config'
+import { checkAndInstallAppInEnvironments } from '../../utils/app-installation'
+import { handleAsyncError as handle } from '../../utils/async'
+import { createPlainClient } from '../../utils/contentful-clients'
+import { ContentTypeApiHelper } from '../../utils/merge/content-type-api-helper'
+import { printChangesetMessages } from '../../utils/merge/print-changeset-messages'
+import { ChangesetItem } from '../../utils/merge/types'
+
+module.exports.command = 'show'
+
+module.exports.desc = 'Show diff between two environments'
+
+module.exports.builder = (yargs: Argv) => {
+  return yargs
+    .usage('Usage: contentful merge show')
+    .option('source-environment-id', {
+      alias: 's',
```

Comment:
As per our recent discussion, we might want to change this to --se to avoid confusion with the -s alias used for space-id in the rest of the CLI

Reply:
@andreascful good point - done

Reply:
It looks like the alias is still -s. Or am I missing something? 🤔

Reply:
Ah sorry, I decided to ship it with a separate PR